e2e/terraform/shared/config/provision-server.sh

```diff
@@ -49,11 +49,14 @@ NOMAD_SRC=/ops/shared/nomad
 NOMAD_DEST=/etc/nomad.d
 NOMAD_CONFIG_FILENAME=$(basename "$NOMAD_CONFIG")
 
-# download
-aws s3 cp "s3://nomad-team-test-binary/builds-oss/nomad_linux_amd64_${NOMAD_SHA}.tar.gz" nomad.tar.gz
-
-# unpack and install
-sudo tar -zxvf nomad.tar.gz -C /usr/local/bin/
+# download, unpack and install if the nomad sha has been set
+# (otherwise it's been installed by the provisioner)
+if [ ${NOMAD_SHA} != "" ]; then
```

Comment:
I'd left this in for backwards compat, but in some testing just now I've realized this is broken if you try to pass a `nomad_version` into the provisioning script. Because this PR was made to a non-master branch I'll just remove this entirely.

Reply:
Done in 4507a7a